privval/file.go

```diff
@@ -198,37 +225,57 @@ func LoadFilePVEmptyState(keyFilePath, stateFilePath string) (*FilePV, error) {
 	return loadFilePV(keyFilePath, stateFilePath, false)
 }
 
+func fileAndContents(path string) (*os.File, []byte, error) {
+	f, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE, 0755)
```

Comment:
I'm not sure I see execute as being needed on this file.

Reply:
Sure I'll set the permissions to be read+write only, I was being generous to the current user.